code/frontend/src/components/ContractorInfo/index.js

```diff
@@ -67,9 +74,22 @@ class ContractorInfoContainer extends React.Component {
     if (typeof contractorData === typeof undefined) {
       contractorData = [];
     }
+    let data = [];
+    if(state.searchvalue == ''){
+      data = contractorData;
+    } else {
+      data = contractorData.filter(contractor => (
+        (contractor['First Name'].toLocaleLowerCase().includes(state.searchvalue.toLocaleLowerCase())) ||
```

Comment:
you can just use `toLowerCase` instead of `toLocaleLowerCase`

Reply:
done